specification/registry/xr.xml

```diff
@@ -2551,6 +2551,28 @@ maintained in the default branch of the Khronos OpenXR GitHub project.
             <member><type>XrActionSet</type> <name>actionSet</name></member>
             <member><type>uint32_t</type>    <name>priorityOverride</name></member>
         </type>
+
+        <!-- XR_MNDX_force_feedback_curl -->
+        <type name="XrForceFeedbackCurlLocationMNDX" category="enum"/>
+
+        <type category="struct" name="XrSystemForceFeedbackCurlPropertiesMNDX" structextends="XrSystemProperties" returnedonly="true">
+            <member values="XR_TYPE_SYSTEM_FORCE_FEEDBACK_CURL_PROPERTIES_MNDX"><type>XrStructureType</type> <name>type</name></member>
+            <member><type>void</type>*    <name>next</name></member>
+            <member><type>XrBool32</type> <name>supportsForceFeedbackCurl</name></member>
+        </type>
+
+        <type category="struct" name="XrForceFeedbackCurlApplyLocationsMNDX">
+            <member values="XR_TYPE_FORCE_FEEDBACK_CURL_APPLY_LOCATIONS_MNDX"><type>XrStructureType</type> <name>type</name></member>
+            <member>const <type>void</type>*                                               <name>next</name></member>
+            <member len="locationCount"><type>XrForceFeedbackCurlApplyLocationMNDX</type>* <name>locations</name></member>
+            <member><type>uint32_t</type>                                                  <name>locationCount</name></member>
```

Comment:
```suggestion
            <member><type>uint32_t</type>                                                  <name>locationCount</name></member>
            <member len="locationCount"><type>XrForceFeedbackCurlApplyLocationMNDX</type>* <name>locations</name></member>
```

This fails ./checkSchematron.sh because the locationCount is after the locations. Is it OK to change the order of those? (otherwise we will have to add this as an exception in the schematron rules.)